repos/ergocub2-design-lowerarm.yml

```
@@ -0,0 +1,14 @@
+ergocub2-design-lowerarm:
+  hsp-iit/sw-dev:
+    type: "group"
+    permissions: "read"
+  
+  ami-iit/mech:
+    type: "group"
+    permissions: "read"
+
+  PasMarra:
+    type: "user"
+    permissions: "read"
+
+  
```

```suggestion
  ami-iit/ergocub:
    type: "group"
    permissions: "read"

  hsp-iit/ergocub:
    type: "group"
    permissions: "read"
```

I reckon this configuration is a better fit.
I double checked the people in the groups and we should be ok.
I'll proceed this way. We can alway fine tune it later.